### ISSUE TITLE
Prompt limit of 125 

### ISSUE THREAD

**Issue by StewieGriffin** [adieyal/sd-dynamic-prompts#45]
I have tested this in several ways.

It seems the current prompt limit when using wildcards is 125 (the value listed on the UI, next to the "Generate" button); after this value, the wildcards stop working.

Or at least, this is happening on my end.

**Comment by adieyal** [comment]
Could you provide a concrete example? I tried this prompt which worked:

    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    one two three four five six seven eight nine ten
    {red|blue}

**Comment by StewieGriffin** [comment]
Weirdly, it's working fine today.
Yesterday it failed all the time when I used any prompt (with Wildcards) above 125 (it would generate an img, but the wildcards were all ignored), and it started working again when the prompt was under 125.
Tested it several times yesterday, even after restarting the webui.

For some reason, it is working fine today, so I'll close this 
This is a great script. Thanks!
